MediaPlayer/MediaControls.module.less

```diff
@@ -0,0 +1,61 @@
+@import "../styles/mixins.less";
+@import "../styles/variables.less";
+@import "../styles/skin.less";
+
+.controlsFrame {
+	position: relative;
+	display: block;
+	text-align: center;
+	direction: ltr;
+	white-space: nowrap;
+	bottom: 24px;
```

Comment:
This makes unnecessary space below the controls. Also, it's not close enough to the slider.

Reply:
I increased the `bottom` value and added a `height: 132px` to remove the unnecessary below space. What do you think?

Reply:
I'm not sure. I'll leave this as is.